graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/processor/EntityMappingModel.java

```diff
@@ -173,14 +173,21 @@ public boolean isFederated() {
         if (partitionKey.isEmpty()) {
           FieldMappingModel firstField = regularColumns.get(0);
           if (TypeHelper.isGraphqlId(firstField.getGraphqlType())) {
+            context.addInfo(
```

Comment:
When this is executed? Is it only once during the init phase? Or for every request (same question for other `info`)

Reply:
`MappingModel` is rebuilt every time we need to refresh the `GraphQL` instance (`SchemaProcessor` is the high-level entry point to do that).

This happens:
- whenever a `deploySchema` mutation is executed (`DeploySchemaFetcherBase`)
- whenever we find out that a new row was inserted in the `graphql_schema` table (`SchemaFirstCache`). We check for every request, but we only rebuild if the version has changed.

Reply:
Ok, I wonder how often in practice this will happen. Won't the `log.info` span the end-user too much?

Reply:
I plan to add a filter to `DeploySchemaResponse.messages`. That would allow you to select warning messages only.